### PR TITLE
Add GetSystemMetrics, EnumDisplayDevices

### PR DESCRIPTION
**Pull request by Green7**
GetSystemMetrics, EnumDisplayDevices and rearrange code to match the order in winuser.h

### CODE REVIEW

File: wrappers/winuser.go [hunk 1]
```diff
@@ -105,26 +106,117 @@ const (
...
-	procOpenInputDesktop = moduser32.NewProc("OpenInputDesktop")
-	procSetThreadDesktop = moduser32.NewProc("SetThreadDesktop")
+	procCloseDesktop       = moduser32.NewProc("CloseDesktop")
+	procEnumDisplayDevices = moduser32.NewProc("EnumDisplayDevicesW")
+	procGetSystemMetrics   = moduser32.NewProc("GetSystemMetrics")
+	procOpenInputDesktop   = moduser32.NewProc("OpenInputDesktop")
+	procSetThreadDesktop   = moduser32.NewProc("SetThreadDesktop")
 )
 
-func CloseDesktop(desktop syscall.Handle) error {
```
Comment (dgolub) [inline wrappers/winuser.go]: Functions should be in alphabetical order.  It's just the constants and structures for which we follow Microsoft's order.

File: wrappers/winuser.go [hunk 2]
```diff
@@ -125,6 +228,24 @@ func CloseDesktop(desktop syscall.Handle) error {
...
+		uintptr(unsafe.Pointer(device)),
+		uintptr(devNum),
+		uintptr(unsafe.Pointer(displayDevice)),
+		uintptr(flags),
+		0,
+		0)
+	return r1 != 0
+}
+
+func GetSystemMetrics(index uint32) int {
```
Comment (dgolub) [inline wrappers/winuser.go]: The type of `index` should be `int32` to match what Microsoft does.  See https://docs.microsoft.com/en-us/windows/desktop/api/winuser/nf-winuser-getsystemmetrics.

File: wrappers/wingdi.go [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+	DeviceName   [32]uint16
+	DeviceString [128]uint16
+	StateFlags   uint32
+	DeviceID     [128]uint16
+	DeviceKey    [128]uint16
+}
+
+const (
+	DISPLAY_DEVICE_ACTIVE           = 0x00000001
+	DISPLAY_DEVICE_ATTACHED         = 0x00000002
```
Comment (dgolub) [inline wrappers/wingdi.go]: It appears that `DISPLAY_DEVICE_ATTACHED` is undocumented and should therefore be omitted.  (If you've found documentation for it somewhere that I missed, please let me know.)

File: wrappers/winuser.go (hunk 2 above)
Comment (Green7) [reply wrappers/winuser.go]: You are right.

File: wrappers/wingdi.go (hunk 3 above)
Comment (Green7) [reply wrappers/wingdi.go]: Indeed is undocumented, so I'm removing it.

**Comment by Green7** [comment]
How about last commit ? Should something be corrected ?

**Pull request merged**